sound/soc/sof/pm.c

```diff
@@ -274,6 +274,8 @@ int snd_sof_dsp_power_down_notify(struct snd_sof_dev *sdev)
 
 int snd_sof_runtime_suspend(struct device *dev)
 {
+	pr_err("%s: ENTER\n", __func__);
```

Comment:
these look like debug left-overs

Reply:
This is the whole point of the patch, to print out the call sequences ;)

Reply:
I will drop this last TEST patch when I move this PR out from draft, I have added it so if anyone is interested to see the pm_runtime sequences can check the prints from the core and from a client.
I needed proof that we have correct sequencing in this regards. This was one of my concern area.

Reply:
I avoid such confusion I typically submit two PRs: one with and one without test commit(s). I link the two and never promote the latter above draft and I close it early. Also provides more "diverse" test coverage immediately.

This is purely about github, I don't know anything about this code change. My 2 cents.

Reply:
@marc-hb , thanks for the hint, I'll do this in the future. This is a draft PR to seek early feedback on the overall direction.